FlipCard.js

```diff
@@ -47,19 +47,24 @@ class FlipCard extends Component{
             friction: this.props.friction,  // Oscillate a lot
         }).start();
     };
-
+    _getOpacity() {
+        if (!this.props.activeOpacity) {
+            return 1;
```

Comment:
I would suggest we return the default of 0.2 and the let the user override the default with 1. Everything else looks good~